src/components/vr_module/include/vr_module/socket_channel.h

```diff
@@ -47,6 +47,7 @@ namespace vr_module {
 class SocketChannel : public Channel {
  public:
   SocketChannel();
+  explicit SocketChannel(net::ConnectedSocket *socket);
```

Comment:
[Is ``int* p;'' right or is ``int *p;'' right?](http://www.stroustrup.com/bs_faq2.html#whitespace)
So, I will use your version for C++ code from now.